resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -231,6 +248,83 @@ class MousePointer {
     }
 }
 
+[DSCResource()]
+class AnimationEffects {
+	#Need to verify the Animation Effects setting toggle on the Accessibility page changes when these are updated, or find the setting to update that also.
+    [DscProperty(Key)] [BinarySettingState] $SmoothScrollListBoxes = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $SlideOpenComboBoxes = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $FadeOrSlideMenusIntoView = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $ShowShadowsUnderMousePointer = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $FadeOrSlideToolTipsIntoView = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $FadeOutMenuItemsAfterClicking = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $ShowShadowsUnderWindows = [BinarySettingState]::KeepCurrentValue
+    [DscProperty(NotConfigurable)] [AnimationEffects] $currentState
+
+    [AnimationEffects] Get() {
+        $this.currentState = [AnimationEffects]::new()
+
+		$this.currentState = "Disabled"
+
+		foreach ($enum in [AnimationEffectsSettings].GetEnumNames()) {
+			$thisState = Get-AnimationState $enum
+			if ($thisState -eq [BinarySettingState]::Enabled) {
+				$this.currentState = "Enabled"

```

Comment:
$this.currentState is an `AnimationEffects` object with each of the DSC properties that you have specified. So you will want to set the enum value to those properties directly rather than converting it to a string. 

For example:

`$this.currentState.SmoothScrollListBoxes = [BinarySettingState]::Enabled`

Reply:
I wasn't sure if `enum`s were allowed there.

Reply:
Please take another look at this, as you need to set the property of the object not the object itself.